.golangci.yml

```diff
@@ -35,12 +32,6 @@ linters:
     # find unused variables, functions, structs, types, etc.
     - unused
 
-    # find unused struct fields
-    - structcheck
-
-    # find unused global variables
-    - varcheck
-
     # parse and typecheck code
```

Comment:
I think its not wise to remove such standard checks as quality of samples repo is being improvised via these

Reply:
varcheck and other two linters are deprecated and replaced by unused linter.